src/test/cc-examples/on_request_headers/http.cc

```diff
@@ -24,6 +24,9 @@ class ExampleContext : public Context {
   explicit ExampleContext(uint32_t id, RootContext *root) : Context(id, root) {}
 
   FilterHeadersStatus onRequestHeaders(uint32_t headers, bool end_of_stream) override;
+  void onDone() override;
+  void onLog() override;
```

Comment:
This is the bottom line of it, the function `onLog` was not defined by the module.
I know too little about this project to suggest a more comprehensive fix or something ...